### DISCUSSION TITLE
part-ii/2018-96/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#97] [Question comments]
Part II 2018 Number Fields: Paper 2, Section II, 20G 

https://questions.tripos.org/part-ii/2018-96/

**Comment by artie2000** [comment]
Don't worry too much about this question -- it's from a year when cyclotomic fields were lectured.
In particular, I think part (c) is not doable in exam conditions with our (2022) tools.
